app/views/users/new.html.erb

```diff
@@ -1 +1,52 @@
-<h1>Welcome to the new user page</h1>
\ No newline at end of file
+<h1>Welcome to the new user page</h1>
+
+<%# <form method='post' action ='/users' accept-charset="UTF-8"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>"> %>
+<%# 
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="email"><br><br>
+
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="username"><br><br>
+
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="password"><br><br>
+
+  <input type="submit">
+
+</form>  %>
```

Comment:
1. In line 3 instead of `action ='/users'` use `action ='"users_path"` this is for make use of users_path helper.